Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -617,7 +616,7 @@ public T TryResolve<T>()
             }
             catch (Exception ex)
             {
-                Logger.LogError(ex, "Error loading assembly {file}", file);
+                Logger.LogError(ex, "Error loading assembly {File}", file);
```

Comment:
What's this change do?

Reply:
Depending on how your logging is set up, the name in between the braces shows up in the logs. PascalCase is the default for these names, while this one was camelBack.
So just a style issue.